frontend/assignments/1/app.js

```diff
@@ -0,0 +1,130 @@
+const button = document.querySelector(".profile-icon");
+
+button.addEventListener("click", () => {
+    console.log("Clicked!!");
+    const leftPanel = document.getElementById("left");
+    leftPanel.style.width = "280px";
+    leftPanel.style.zIndex = "1";
+    leftPanel.style.position="fixed";
+})
+
+
+// Function to create a new post element
+function createPostElement(tweetText) {
+    // Replace hashtags with span elements to style them
+    tweetText = tweetText.replace(/#(\w+)/g, '<span class="hashtag">#$1</span>');
+    // Create elements
+    const postDiv = document.createElement("div");
+    postDiv.classList.add("each-post");
+    const postContent = `
+        <img src="images/profile pic.png" alt="" id="profile-image" />
+        <div class="post-content-right">
+        <div class="post-upper">
+            <div class="post-owner-handle">
+            <p>Nitesh Gupta <span>@nik_hck</span></p>
+            </div>
+            <a href="">
+            <img src="icons/three_dots.svg" alt="" id="three-dots" />
+            </a>
+        </div>
+        <div class="post-middle">
+            <p>${tweetText}</p>
+        </div>
+        <div class="post-lower">
+            <div class="post-actions-links">
+            <a href="">
+                <img src="icons/comment.svg" alt="" />
+            </a>
+            <a href="">
+                <img src="icons/retweet.svg" alt="" />
+            </a>
+            <button class="like-button">
+                <img src="icons/heart.svg" alt="" id="heart" />
+                <span class="like-count"></span>
+            </button>
+            <a href="">
+                <img src="icons/stats.svg" alt="" />
+            </a>
+            <a href="">
+                <img src="icons/bookmark-post.svg" alt="" />
+            </a>
+            </div>
+            <a href="">
+            <img src="icons/share.svg" alt="" />
+            </a>
+        </div>
```

Comment:
ideally lets put these in html and just set visibility none or display none on the className or id